packit/command_handler.py

```diff
@@ -168,20 +181,21 @@ def run_command(
 
         logger.info(f"Running command: {' '.join(command)}")
 
-        success, out = True, ""
+        out = ""
         try:
             out = self.sandcastle.exec(command=command, env=env, cwd=cwd)
 
             logger.info(f"Output of {command!r}:")
-            # out = 'make po-pull\nmake[1]: Entering directory \'/sand
-            for output_line in out.split("\n"):
-                if output_line:
-                    logger.info(output_line)
+            self._print_logs(out)
         except SandcastleCommandFailed as ex:
-            # command failed in the sandcastle, return the logs anyways
-            success, out = False, ex.output
+            # TODO: Maybe consider logging as an error? Would spam Sentry though
```

Comment:
you're right: we may want to know when user commands fail, but we certainly don't *need* to know